bridgesample/src/main/java/com/livefront/bridgesample/scenario/activity/LargeDataActivity.kt

```diff
@@ -34,9 +39,17 @@ class LargeDataActivity : BridgeBaseActivity() {
     }
 
     companion object {
-        fun getNavigationIntent(context: Context) = Intent(
+        private const val INFINTE_BACKSTACK_KEY = "infinite-backstack"
+
+        fun getInfiniteBackstack(
+                activity: LargeDataActivity
+        ): Boolean = activity
+                .intent
+                .getBooleanExtra(INFINTE_BACKSTACK_KEY, false)
+
+        fun getNavigationIntent(context: Context, infiniteBackstack: Boolean) = Intent(
                 context,
                 LargeDataActivity::class.java
-        )
+        ).apply { putExtra(INFINTE_BACKSTACK_KEY, infiniteBackstack) }
```

Comment:
Let's format this as

```kotlin
fun getNavigationIntent(
    context: Context,
    infiniteBackstack: Boolean
) = Intent(context,  LargeDataActivity::class.java).apply {
    ...
}
```

Reply:
Also, in general we use an "arguments" class to encapsulate all non-context arguments here . See for example : 
https://github.com/livefront/bridge/blob/master/bridgesample/src/main/java/com/livefront/bridgesample/scenario/activity/FragmentContainerActivity.kt

This just helps if we ever add another parameter so that we don't need more Intent / Bundle keys , more methods to grab the data, etc. So let's update to use a `LargeDataActivityArguments` class or something.

Reply:
Done.